src/theme/border/theme-helper.ts

```diff
@@ -10,31 +10,35 @@ const BorderSpec: ThemeCollectionSpec<BorderCollectionMap, BorderScheme> = {
     },
     defaultValue: "lifesg",
 };
+// to check if hex colour
+const isHexColor = (color: string): boolean => {
```

Comment:
is there a need to check for hex format?

Reply:
The reason I did that is if I were to use the semantic colour straight then there is no need to encode it because the return is a hex string but when I were to pass in arguments like (1, "red") for the border then we need to encode the red to fit the svg. So thats why i added a check to see if its a hex.